hooks/collector.py

```diff
@@ -219,13 +248,16 @@ def _collect_folder(self, parent_item, folder):
         :returns: The item that was created
         """
 
+        if not self.IMAGE_EXTENSIONS_LIST:
+            self.IMAGE_EXTENSIONS_LIST = self._build_image_extensions_list()
```

Comment:
a more pythonic approach would be to have a memoize decorator.

Reply:
the method handles its own caching now.